docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1318,6 +1318,58 @@ The output of the query has the following columns:
   - File metrics in human-readable form.
 :::
 
+##### `$entries` table
+
+The `$entries` table provides all the table's current manifest entries for both 
+data and delete files.
+
+To retrieve the information about the entries of the Iceberg table
+`test_table`, use the following query:
+
+```
+SELECT * FROM "test_table$entries"
+```
+
+```text
```

Comment:
I think we can cut this output down .. for data_file and readable_metrics just start with the first bit and then use ...

Reply:
and above the table say something like

Abbreviated sample output: